src/registrar/views/domain_request.py

```diff
@@ -392,6 +394,7 @@ def get_context_data(self):
             "is_federal": self.domain_request.is_federal(),
             "modal_button": modal_button,
             "modal_heading": modal_heading,
+            "has_profile_feature_flag": flag_is_active(self.request, "profile_feature"),
```

Comment:
This option is temporary.  At some point, this will no longer be an option, and will be turned on in all cases.  So, I don't think necessary at this point to invest in exercising this option globally.